pkg/lib/env.go

```diff
@@ -29,6 +30,13 @@ const (
 	MaxCreateVMsOnProviderEnv     = "MAX_CREATE_VMS_ON_PROVIDER"
 	DefaultMaxCreateVMsOnProvider = 80
 
+	// PrivilegedUsersEnv is the key for the environment variable
+	// containing a set of privileged users. Currently, users pointed by
```

Comment:
The word "currently" might cause confusion because the env var privileged users are included only if the backup & restore FSS are enabled (which is not atm). Could you make this clearer in the comment, or remove the FSS check from the `IsPrivilegedAccount` function?

Reply:
Modified the text to indicate that the privileged users are only allowed when the FSS is enabled.  I think we need to keep the FSS check around to ensure that net-new changes are guarded by an FSS.